### ISSUE TITLE
RTL lits and headings problems

### ISSUE THREAD

**Issue by osman1963** [zim-desktop-wiki/zim-desktop-wiki#1728]
First of all, thanks for the great app, really love it.

I was hoping you can fix the following bugs soo, as they hinder my use of the app.
These bugs occur when switching to an RTL language, Arabic in my case.
I'm using the app on Windows 10, 64 bit, the latest version 7.4.2

1. Pressing tab at the beginning of the line doesn't do anything, however pressing it after typing a few letters works as expected.
2. The lists (both ordered and unordered) can't be nested. Pressing tab does nothing.
3. heading 3 and 5 replaces the text with squares with numbers inside. This happens if you type '=' signs, or choose Heading 3 or Heading 5 from the formatting bar.
4. When pressing Enter,  for a new line, the cursor is always at the left.

Thanks in advance

**Comment by jaap-karssenberg** [comment]
Yes would really want to have the application RTL compatible. However since
I personally don't read and use any RTL script I hope someone with more
experience with using RTL in computer programs can help out fixing this.

Op zo 7 nov. 2021 14:27 schreef osman1963 ***@***.***>:

**Comment by osman1963** [comment]
hope so, and thanks so much for your prompt answer :)
